deel-2-flexbox-layout/opdracht-5/styles/styles.css

```diff
@@ -1 +1,46 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+body {
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+main {
+    display: flex;
+    flex-basis: auto;
+    flex-grow: 1;
+    flex-shrink: 1;
+}
+
+header {
+    flex-basis: 200px;
+    flex-grow: 0;
+    flex-shrink: 0;
+}
+
+footer {
+    flex-basis: 200px;
+    flex-grow: 0;
+    flex-shrink: 0;
+}
+
+nav {
+    flex-basis: 20%;
+    flex-grow: 0;
+    flex-shrink: 0;
+    order: 1;
```

Comment:
De uitwerking geeft aan dat als je de order naar -1 deed op de nav, je het probleem ook had kunnen oplossen.
Maar dit is net zo prima, ziet er netjes uit. Alleen volegorde 👍